src/constants/imageData.ts

```diff
@@ -0,0 +1,13 @@
+const imageUrls: string[] = [
+    '/src/assets/card-images/card-image1.png',
+    '/src/assets/card-images/card-image2.png',
+    '/src/assets/card-images/card-image3.png',
+    '/src/assets/card-images/card-image4.png',
+    '/src/assets/card-images/card-image5.png',
+    '/src/assets/card-images/card-image6.png',
+    '/src/assets/card-images/card-image7.png',
+    '/src/assets/card-images/card-image8.png',
+    '/src/assets/card-images/card-image9.png',
+  ];
+  
+  export default imageUrls;
```

Comment:
Rename it to more specific name, we will have more image urls, you don't want to mix them